packages/instant/src/util/balance.ts

```diff
@@ -0,0 +1,13 @@
+import { BuyQuote } from '@0x/asset-buyer';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import * as _ from 'lodash';
+
+export const balanceUtil = {
+    hasSufficientFunds: async (takerAddress: string | undefined, buyQuote: BuyQuote, web3Wrapper: Web3Wrapper) => {
+        if (_.isUndefined(takerAddress)) {
+            return false;
+        }
+        const balanceWei = await web3Wrapper.getBalanceInWeiAsync(takerAddress);
+        return balanceWei >= buyQuote.worstCaseQuoteInfo.totalEthAmount;
```

Comment:
we might want to keep this generalized by passing in a BigNumber representing the "sufficient" balance. Also the name of the method could be improved since fund can refer to eth or tokens. What about `hasSufficientEth`?

Reply:
I like the name change.  I'd prefer to keep the logic as-is, as this helper function would hardly do anything if you sent in the balance.  I'd like this function to be helpful if we validate balance on typing (instead of just when clicking 'Buy') in the future